### DISCUSSION TITLE
PPOCRLabel 的检测模型是否支持PaddleDetection训练导出的模型？

### DISCUSSION THREAD

**Discussion by Eason-shu** [PaddlePaddle/PaddleOCR#13793] [Q&A]
PPOCRLabel  的原有的检测模型，满足不了我的要求，于是我PaddleDetection训练了检测模型，我想是否能够替换PPOCRLabel的检测模型，代替人工标注

**Comment by GreatV** [comment]
你自己写个脚本转换一下就好了。
